src/open_inwoner/conf/parts/maileditor.py

```diff
@@ -233,7 +233,7 @@ def _readfile(name):
                 "description": _("Name of the site"),
             },
             {
-                "name": "contactpage",
+                "name": "contact_page",
```

Comment:
This would need 'contact_phonenumber' as well, but I can add that later myself, since some parts of the templates will be changing a bit.